specs/experimental/l2_blob_tx.md

```diff
@@ -0,0 +1,43 @@
+# L2 Blob Transaction
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [L2 Blob Transaction](#l2-blob-transaction)
+  - [Motivation](#motivation)
+  - [How It Works](#how-it-works)
+  - [Reference Implementation](#reference-implementation)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Motivation
+
+To make it seamless for dapps that's depending on blob transaction to migrate from L1 to L2, it's important that L2 also supports blob transaction. This will also make L3 more uniform since they can also use blob transaction for batch submitting, just like L2.
+
+## How It Works
+
+The changes include two parts: optimism and op-geth.
+
+The optimism part mainly includes:
+1. Add `L2BlobConfig` field to `rollup.Config` so that if configured, L2 blob tx will be supported and optionally the blob can be stored to a [DA provider](https://github.com/ethstorage/da-server) by sequencer.
```

Comment:
```suggestion
1. Add `L2BlobConfig` field to `rollup.Config` so that if configured, L2 blob tx will be supported and optionally the blob can be stored to a [DA provider](https://github.com/ethstorage/da-server) by the sequencer.
```